spec/lib/terraforming/resource/datadog_monitor_spec.rb

```diff
@@ -0,0 +1,197 @@
+require "spec_helper"
+
+module Terraforming
+  module Resource
+    describe DatadogMonitor do
+      let(:client) { instance_double(Dogapi::Client).as_null_object }
+
+      let(:monitors) do
+        [ "200",
+          [
+            {
+              "tags"=>[],
+              "deleted"=>nil,
+              "query"=>"avg(last_1h):avg:aws.xxxx > 15000",
+              "message"=>"@slack-team-xxxx",
```

Comment:
__[rubocop]__
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->